skimage/filters/thresholding.py

```diff
@@ -1064,6 +1064,17 @@ def threshold_multiotsu(image, classes=3, nbins=256):
     between the desired classes. Based on the Multi-Otsu approach by
     Liao, Chen and Chung.
 
+    This implementation relies on a Cython function whose complexity
+    is O(combinations * variance_calc), and depends on the number of
+    bins and classes:
+
+    >>> import math
+    >>> combinations = nbins**(classes-1)/math.factorial(classes-1)
+    >>> variance_calc = classes + 1
+
+    Then, the required calculation time grows exponentially according
+    to the number of classes desired.
+
```

Comment:
I think this should be straight-up math notation rather than Python. I suggest

```
whose complexity is :math:`O\left(\frac{Ch^{C-1}}{(C-1)!}\right)` where :math:`h` is
the number of histogram bins and :math:`C` is the number of classes desired.
```

(If I remember my complexity analysis correctly, the +1 can be safely left out of the expression.)

Reply:
You might need to double-check my sphinx-math expression. =)

Reply:
Awesome! I'll do that. (never learned complexity analysis :laugh:)